src/components/Quiz/useQuizStore.ts

```
@@ -726,6 +726,13 @@ export const useQuizStore = create<QuizState>()(
             (answer) => answer.questionId === currentQuestion.id
           );
 
+          // Retorna undefined se a resposta está vazia (não respondida)
+          if (userAnswer && 
```

ESLint

[error] Replace `userAnswer·&&·` with `⏎············userAnswer·&&`

(prettier/prettier)